### ISSUE TITLE
[Feature Request] Add ExFat Format As An Option

### ISSUE THREAD

**Issue by annoyingduck** [linuxmint/mintstick#60]
Is it possible to add exfat as a format option? Mintstick is my go to for everything relating to a usb, and currently the best gui option for formatting in exfat is to use the gnome-disk-utility (or booting into Windows - Vomit!). Having the simple option within Mintstick would really be nice. There is an application made specifically for Elementary OS that has an exfat format option which leads me to believe that this can be implemented into Mintstick.  Here's the app here on Github (it's just called formatter): https://github.com/Djaler/Formatter

**Comment by xenopeek** [comment]
Implemented in Linux Mint 19 release.

**Comment by RoyiAvital** [comment]
Will `minstick` be able to create Windows 10 Bootable USB (To install Windows) from Windows 10 ISO file?

**Comment by logix2** [comment]
I'm not seeing exFAT in USB Stick Formatter as an option using the latest mintStick. Did anyone got this to work?

**Comment by xenopeek** [comment]
exFAT support was added but due to mixup on my branch it wasn't added as a selectable option in the UI in the release. Fixed with PR #66.

You can add the missing line of code on your system by running the following command from the terminal (whitespace in this command is important so copy it verbatim):
```
sudo sed -i '/"fat32"/a \            self.fsmodel.append(["exfat", "exFAT",      15,        False,               False])' /usr/lib/mintstick/mintstick.py
```

Or alternatively manually edit the file /usr/lib/mintstick/mintstick.py, search for "self.fsmodel.append" and after the fat32 line add the following line line (make sure the line has the same number of leading spaces as the lines above and below it):
```
            self.fsmodel.append(["exfat", "exFAT",      15,        False,               False])
```
